code/script.js

```diff
@@ -0,0 +1,79 @@
+const USER = 'Groovedharry'
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`;
+const USER_API = `https://api.github.com/users/${USER}`;
+const projectsContainerXXYT = document.getElementById('profileInfo');
+const projectsContainer = document.querySelector("#projects");
+
+
+const getUser = () => {
+    fetch(USER_API)
+        .then((response) => response.json())
+        .then((data) => {
+            profile.info.innerHTML += `
+        <h2> Profile info </h2>
+        <img src ="https://avatars.githubusercontent.com/u/83465217?v=4" alt="profile pic">
+        <h4> ${data.name}</h4>
+        <h4> >a href =${data.html_url}>${data.login}</h4>
+        `;
+        });
+};
+
+// Repos //
+const getRepos = () => {
+    fetch(REPOS_URL)
+        .then((response) => {
+            return response.json();
+        })
+        .then((json) => {
+            const forkedRepos = json.filter((project) => project.fork && project.name.startsWith("project-"));
+            console.log(forkedRepos)
```

Comment:
I would remove this console log before pushing the final result, but since you have written that you arent entirely done with the project I understand that you left it =)